devnet-sdk/system2/el.go

```diff
@@ -0,0 +1,64 @@
+package system2
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum-optimism/optimism/op-service/client"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/sources"
+)
+
+type EthClient interface {
+	SendTransaction(ctx context.Context, tx *types.Transaction) error
+	// more methods may be added
+}
+
+type ELNode interface {
+	Common
+	ChainID() eth.ChainID
+	EthClient() EthClient
+}
+
+type ELNodeConfig struct {
+	CommonConfig
+	Client  client.RPC
+	ChainID eth.ChainID
+}
+
+type rpcELNode struct {
+	commonImpl
+
+	cl      client.RPC
```

Comment:
`client` maybe? `cl` can sound confusing in the context of `el`/`cl` saturated code